server/rpm/seqno

```diff
@@ -0,0 +1 @@
+91
```

Comment:
Is `server/rpm/seqno` really a file we want to commit to the tree?

Reply:
No.